Pod/Classes/ProductRecommendationModel.swift

```diff
@@ -51,7 +52,7 @@ public class ProductRecommendationModel: NSObject, NSCopying {
                 imageS: String?, imageM: String?, imageL: String?, imageXL: String?, category: [String]?, categories: [String]?,
                 brand: String?, price: NSNumber?, oldPrice: NSNumber?, gender: String?, colors: [String]?, sizes: [String]?,
                 labels: [String]?, noUpdate: Bool?, params: [String: AnyObject]?, priceText: String?, oldPriceText: String?,
-                language: String?, currency: String?, specialPrice: NSNumber?, specialPriceText: String?) {
+                language: String?, currency: String?, specialPrice: NSNumber?, specialPriceText: String?, region: String?) {
```

Comment:
[Product object](https://github.com/segmentify/segmentify-v3/blob/dev/commons/src/main/java/com/segmentify/lotr/commons/model/product/Product.java) does not have region field. I think we do not need this change.